src/native/corehost/hostpolicy/deps_format.cpp

```diff
@@ -180,31 +180,148 @@ void deps_json_t::reconcile_libraries_with_targets(
     }
 }
 
-// Returns the RID determined (computed or fallback) for the platform the host is running on.
-pal::string_t deps_json_t::get_current_rid(const rid_fallback_graph_t& rid_fallback_graph)
+namespace
 {
-    pal::string_t currentRid = get_current_runtime_id(false /*use_fallback*/);
+    const pal::char_t* s_host_rids[] =
+    {
+#if defined(TARGET_WINDOWS)
+        _X("win"),
+#elif defined(TARGET_OSX)
+        _X("osx"),
+        _X("unix"),
+#elif defined(TARGET_LINUX_MUSL)
+        _X("linux-musl"),
+        _X("linux"),
+        _X("unix"),
+#elif defined(TARGET_ANDROID)
+        _X("linux-bionic"),
+        _X("linux"),
+        _X("unix"),
+#else
+        _X("linux"),
```

Comment:
Maybe better to keep it scalable:
```suggestion
        _STRINGIFY(FALLBACK_HOST_OS),
```
It will cover BSD and SunOS likes.